themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl

```diff
@@ -54,16 +54,8 @@
       {/if}
       {if $contact_infos.email && $display_email}
         <br>
-        {* [1][/1] is for a HTML tag. *}
-        {l
-          s='Email us: [1]%email%[/1]'
-          sprintf=[
-            '[1]' => '<a href="mailto:'|cat:$contact_infos.email|cat:'" class="dropdown">',
-            '[/1]' => '</a>',
-            '%email%' => $contact_infos.email
-          ]
-          d='Shop.Theme.Global'
-        }
+          {l s='Email us:' d='Shop.Theme.Global'}
+          {mailto address={$contact_infos.email} encode="javascript"}
```

Comment:
```suggestion
          {mailto address=$contact_infos.email encode="javascript"}
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17562/commits/d5f5be793c60dd6219fa5977f9595289dfa4063c